### DISCUSSION TITLE
Error popup

### DISCUSSION THREAD

**Discussion by ScionoicS** [Nevysha/Cozy-Nest#76] [Ideas]
Anytime i open the img2img tab, this totally irrelevant error about a handler not being attached pops up. It eats the entire screen and obnoxiously demands your attention until it's dealt with.  It doesn't even display the actual error as the up most information and instead has it near the bottom. On a 1440p monitor, i had to scroll past a huge amount of technical data dumps for logging  purposes, to get to the actual relevant error message. It's a mundane message and is meaningless to an end user. As far as i can tell, the user experience isn't harmed at all by this event handler not hooking correctly. 

A good idea for a UI focused extension would be not to display errors this way. Especially simple and mundane errors that don't effect the end user experience.

**Comment by ScionoicS** [comment]
https://github.com/Nevysha/Cozy-Nest/commit/40d5eb2fc2c44bdf0447f077b9a283189a8ddddd

That was fast

**Reply by ScionoicS** [reply]
oh it was committed before my post was made! carry on then. nothing to see here

**Comment by Nevysha** [comment]
This error popup is mostly usefull for me to troubleshot and help user when they face issues. Btw it was a bit to greedy but should be better. Also you can disable it.

> Cozy Nest error popup should now only be displayed if filename contains Cozy Nest
https://github.com/Nevysha/Cozy-Nest/releases/tag/v2.0.2

![image](https://github.com/Nevysha/Cozy-Nest/assets/122687716/cc420abe-f7ba-444e-b0d7-80ca9e4596ab)


![image](https://github.com/Nevysha/Cozy-Nest/assets/122687716/87af1268-06e9-4eca-96aa-fc124db037ce)
